lib/mongoidable/document_extensions.rb

```diff
@@ -7,7 +7,8 @@ module DocumentExtensions
 
     included do
       ability_class = Mongoidable.configuration.ability_class
-      raise TypeError unless ability_class == Mongoidable::Ability || ability_class.superclass == Mongoidable::Ability
+      raise TypeError, "Mongoidable::Document can only be included in a Mongoid::Document" unless
```

Comment:
The equality checks here were failing as the class object did not match. During the rails reload we ended up with 2 class constants that were not equal. Compare using the class names